docs/michelsonmap.md

```diff
@@ -15,7 +15,7 @@ Taquito reads maps in the storage of smart contracts and translates them into an
 - _The key/value methods_: they allow you to manipulate the keys and values in the map
 - _The update methods_: they transform the map itself, for example, by deleting elements or clearing out the map entirely.
 
-This tutorial uses a [simple smart contract deployed on kathmandunet](https://better-call.dev/kathmandunet/KT1C6YRpA7puibK8by6RrV5uMg6LBBiHauTP/operations) with a map that contains addresses as keys and tez as values. We will use all the methods available in Taquito's `MichelsonMap` to check the map, extract values and modify them!
+This tutorial uses a [simple smart contract deployed on ghostnet](https://better-call.dev/ghostnet/KT1C6YRpA7puibK8by6RrV5uMg6LBBiHauTP/operations) with a map that contains addresses as keys and tez as values. We will use all the methods available in Taquito's `MichelsonMap` to check the map, extract values and modify them!
```

Comment:
This contracts address needs to be updated to reflect the one one on Ghostnet

Reply:
Yes, it is the primary job to get all the contracts replaced with the correct new ones.